data_science/health_monitoring_dashboard/healthdashboard.ipynb

```diff
@@ -0,0 +1,334 @@
+    "    'padding': '30px',\n",
+    "    'background': 'linear-gradient(135deg, #f7f9fc, #e3e7ed)',\n",
+    "    'box-shadow': '2px 2px 10px rgba(0,0,0,0.1)',\n",
+    "    'border-radius': '10px'\n",
+    "})\n",
+    "\n",
+    "# Gauge charts page\n",
+    "gauge_charts_layout = html.Div([\n",
+    "    html.H3(\"Gauge Charts for Key Health Metrics\", style={\n",
+    "        'text-align': 'center',\n",
+    "        'margin-bottom': '20px',\n",
+    "        'font-family': 'Georgia, serif',\n",
+    "        'font-size': '28px',\n",
+    "        'color': '#2C3E50',\n",
+    "        'text-transform': 'uppercase',\n",
+    "        'letter-spacing': '2px',\n",
+    "        'border-bottom': '2px solid #CCC',\n",
+    "        'padding-bottom': '10px'\n",
+    "    }),\n",
+    "    html.Div([\n",
+    "        dcc.Graph(figure=gauge_fig_heart),\n",
+    "        dcc.Graph(figure=gauge_fig_oxygen),\n",
+    "        dcc.Graph(figure=gauge_fig_hydration)\n",
+    "    ], style={'display': 'flex', 'flex-direction': 'column', 'align-items': 'center'})\n",
+    "], style={\n",
+    "    'width': '80%',\n",
+    "    'margin': '0 auto',\n",
+    "    'padding': '30px',\n",
+    "    'background': 'linear-gradient(135deg, #f7f9fc, #e3e7ed)',\n",
+    "    'box-shadow': '2px 2px 10px rgba(0,0,0,0.1)',\n",
+    "    'border-radius': '10px'\n",
+    "})\n",
+    "\n",
+    "# Initialize the Dash app\n",
+    "app = dash.Dash(__name__)\n",
+    "\n",
+    "# Defining the layout for the dashboard with improved styles and the new healthy range page\n",
+    "app.layout = html.Div([\n",
+    "    html.H1(\"Health Monitoring Dashboard\", style={\n",
+    "        'text-align': 'center', \n",
+    "        'font-family': 'Georgia, serif',\n",
+    "        'font-size': '36px', \n",
+    "        'color': '#34495E',\n",
+    "        'padding-top': '20px',\n",
+    "        'padding-bottom': '20px',\n",
+    "        'letter-spacing': '1.5px',\n",
+    "        'border-bottom': '2px solid #CCC'\n",
+    "    }),\n",
+    "    dcc.Tabs([\n",
+    "        dcc.Tab(label='Summary', children=[summary_layout], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='Comparison with Healthy Ranges', children=[healthy_comparison_layout], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='Gauge Charts', children=[gauge_charts_layout], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='Time-Series Analysis', children=[time_series_summary], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='Box Plot: Heart Rate by Fall Detection', children=[box_plot_summary], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='HRV Distribution', children=[hrv_summary], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='Room vs Body Temperature', children=[temperature_summary], style={'padding': '20px'}),\n",
+    "        dcc.Tab(label='Correlation Heatmap', children=[heatmap_summary], style={'padding': '20px'}),\n",
```

Comment:
Can you please check the following errors in these lines:
* Undefined variable 'time_series_summary' (undefined-variable)
* Undefined variable 'box_plot_summary' (undefined-variable)
* Undefined variable 'hrv_summary' (undefined-variable)
* Undefined variable 'temperature_summary' (undefined-variable)
* Undefined variable 'heatmap_summary' (undefined-variable)